includes/Model/Story.php

```diff
@@ -73,6 +80,48 @@ class Story {
 	 */
 	protected $poster_square;
 
+	/**
+	 * Height for displaying story.
+	 *
+	 * @var int
+	 */
+	protected $height;
+
+	/**
+	 * Width for displaying story.
+	 *
+	 * @var int
+	 */
+	protected $width;
+
+	/**
+	 * Classes for story.
+	 *
+	 * @var string
+	 */
+	protected $classes;
```

Comment:
Those are concerns of the renderer, not of the story itself

Reply:
Agree!
Changed it so that Renderer should take care of dimensions.